lighthouse-core/runner.js

```diff
@@ -134,14 +136,21 @@ class Runner {
             a => Aggregate.aggregate(a, runResults.auditResults));
         }
 
+        let categories = [];
+        if (config.report) {
+          const reportGenerator = new ReportGeneratorV2();
+          categories = reportGenerator.generateReportJson(config, resultsById).categories;
+        }
+
         return {
           lighthouseVersion: require('../package').version,
           generatedTime: (new Date()).toJSON(),
           initialUrl: opts.initialUrl,
           url: opts.url,
-          audits: formattedAudits,
+          audits: resultsById,
           artifacts: runResults.artifacts,
           runtimeConfig: Runner.getRuntimeConfig(opts.flags),
+          categories,
```

Comment:
should we put this inside a `report` object on the result payload?  or title it `reportCategories` to communicate clearly that it's just presentational?